extensions/replication/ReplicationQueuePopulator.js

```diff
@@ -78,6 +78,31 @@ class ReplicationQueuePopulator extends QueuePopulatorExtension {
                      `${queueEntry.getBucket()}/${queueEntry.getObjectKey()}`,
                      JSON.stringify(publishedEntry));
     }
+
+    /**
+     * Filter if the entry is considered a valid master key entry.
+     * There is a case where a single null entry looks like a master key and
+     * will not have a duplicate versioned key. They are created when you have a
+     * non-versioned bucket with objects, and then convert bucket to versioned.
+     * If no new versioned objects are added for given object(s), they look like
+     * standalone master keys. The `isNull` case is undefined for these entries.
+     * Null versions which are objects created after suspending versioning are allowed,
+     * these only have a master object that has an internal versionId and a 'isNull' flag.
+     * @param {ObjectQueueEntry} entry - raw queue entry
```

Comment:
(now raw since we parsed it and transformed it into an `ObjectQueueEntry`)

```suggestion
     * @param {ObjectQueueEntry} entry - queue entry
```